src/linux_ip_socket/linux_ip_socket.cc

```diff
@@ -265,7 +309,7 @@ linux_ip_socket_private_data::read_data_or_disconnect(pollfd* table)
         table[0].events = 0;
         return false;
     } else if(recv_result == RECV_CONNECTION_SHUTDOWN) {
-        close(table[0].fd);
+        // close(table[0].fd);
```

Comment:
Why? Also, maybe remove?

Reply:
This shouldn't be uncommented nor removed. The listening side waits for messages and if the sending side attempts to close connection, the connection shall be closed. In case of reusing socket for send, this changes nothing.